### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5970
Fixed Powershell

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ When this change takes effect, meeting recordings will be stored by default in t
 To enable recordings in region the Global policy, use the following cmdlet:
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:113]:
```suggestion
To enable recordings in-region in the Global policy, use the following cmdlet:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - additional edit, please.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**